zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/DefaultExporterResourceProvider.java

```diff
@@ -448,6 +449,7 @@ private void addAuditLogHandlers(final AuditLogConfiguration auditLog) {
       final var indexName = (indexDescriptors.get(AuditLogTemplate.class).getFullQualifiedName());
 
       AuditLogHandler.builder(indexName, auditLog)
+          .addHandler(new ProcessInstanceCancelAuditLogTransformer())
           .addHandler(new ProcessInstanceModificationAuditLogTransformer())
           .addHandler(new BatchOperationCreationAuditLogTransformer())
           .addHandler(new BatchOperationLifecycleManagementAuditLogTransformer())
```

Comment:
🙃  We can sort these alphabetically to make it easier to code navigation and readability in the future.